frontend/dispatcher.lua

```diff
@@ -312,8 +313,10 @@ function Dispatcher:getNameFromItem(item, location, settings)
     if location[settings] ~= nil and location[settings][item] ~= nil then
         amount = location[settings][item]
     end
-    if amount == nil or amount == 0 then
-        amount = "X"
+    if amount == nil
+        or (amount == 0 and settingsList[item].category == "incrementalnumber")
+    then
+        amount = C_("Number placeholder", "#")
```

Comment:
A comment on `Go %1 pages`, not sure otoh if that's related to this specifically but it does bring up a point about not being too clever with constructing strings programmatically.
![Screenshot_2020-08-05_12-07-02](https://user-images.githubusercontent.com/202757/89400464-38c56600-d714-11ea-9c84-c132d902e405.png)

Reply:
this does not change that.

I see no simple way to fix that as we don't know the value originally, and would need to return a function for the title instead of just a string.

Reply:
> and would need to return a function for the title instead of just a string.

I'll take your word for it that it's somewhat complicated since I don't want to look into it right now (or this weekend at 36°…), but that part sounds rather easy. ;-)